pkg/chains/signing.go

```diff
@@ -40,19 +41,15 @@ type Signer interface {
 	Sign(ctx context.Context, obj objects.TektonObject) error
 }
 
-type MetricsRecorder interface {
-	RecordCountMetrics(ctx context.Context, MetricType string)
-}
```

Comment:
This interface should is implemented by `pipelinerunmetrics.Recorder` and `taskrunmetricsRecorder`. So it's been moved to `pkg/metrics/common.go`, and the implementations can now enforce at the type level that they _do_ implement the interface without a circular import...